README.md

```diff
@@ -16,7 +16,7 @@ License URI: https://opensource.org/licenses/GPL-2.0
 
 ## Building
 
-$ zip -r v2.0.0.0.zip ./localiq_joomla_30_extension  -x *.git*
+$ zip -r localiq_plugin_v2.1.0.0.zip ./localiq_joomla_30_extension -x ./localiq_joomla_30_extension/.git/\*
```

Comment:
The project name has been changed. Should we update the directory name here to match it?

Reply:
Yes, done.